drivers/storage/portworx/component/autopilot.go

```diff
@@ -316,6 +316,14 @@ func (c *autopilot) createClusterRole() error {
 		},
 		)
 	}
+	if pxutil.IsOpenshift(cluster) {
```

Comment:
Let's add this irrespective of whether this is OCP or not. Some other env in future might need this.

Reply:
Sure, thanks. I added this resource to the group where other autopilot resources stay in and fixed the UT yamls